### DISCUSSION TITLE
Custom init function

### DISCUSSION THREAD

**Discussion by rickywu** [ayamir/nvimdots#1037] [General]
```
	if global.is_windows then
		vim.cmd([[
            cd D:\Work
            let g:sqlite_clib_path = 'D:\DevTools\Neovim\lib\sqlite3.dll'
        ]])
	end
	if global.is_wsl then
		vim.cmd([[cd /mnt/d/Work]])
	end
```

I want to custom init.lua like this in user settings, but not find example in usage, can you help?

**Comment by ayamir** [comment]
We didn't provide this ability. You can just use this way if it works.

**Comment by rickywu** [comment]
if global.is_windows then
    vim.cmd([[cd D:\Work]])
end
if global.is_wsl then
    vim.cmd([[cd /mnt/d/Work]])
end

I move to user/settings.lua, works well
